docs/cugraph/source/graph_support/algorithms/cpp_algorithms/centrality_cpp.md

```diff
@@ -0,0 +1,81 @@
+# Centrality algorithms
+cuGraph Pagerank is implemented using our graph primitive library
+
+## Pagerank
+
+The unit test code is the best place to search for examples on calling pagerank.
+
+ * [SG Implementation](https://github.com/rapidsai/cugraph/blob/main/cpp/tests/link_analysis/pagerank_test.cpp)
+ * [MG Implementation](https://github.com/rapidsai/cugraph/blob/branch-24.06/cpp/tests/link_analysis/mg_pagerank_test.cpp)
```

Comment:
Do we want to point to a particular branch?  That seems like we would need to update every release.